### ISSUE TITLE
Dropout as Evolution noise?

### ISSUE THREAD

**Issue by jamesal1** [jamesal1/NESgpu#2]
I wrote in the description that inference mode (WX) is faster than any possible training mode layer(WX+EX), but this isn't strictly true. If we use dropout then it's not necessary to compute activations for dropped neurons. It should be possible to train a neural network purely by dropping neurons. Depending on implementation, this should either be almost as fast as inference or faster.
